test/testinput/bufr_ncep_prepbufr_adpupa.py

```diff
@@ -53,50 +53,56 @@ def test_bufr_to_ioda(DATA_PATH, OUTPUT_PATH, date):
 
     # Use the ResultSet returned to get numpy arrays of the data
     # MetaData
-    cat = r.get('prepbufrDataLevelCategory', 'prepbufrDataLevelCategory')
-    lat = r.get('latitude', 'prepbufrDataLevelCategory')
-    lon = r.get('longitude', 'prepbufrDataLevelCategory')
+    cat = r.get('prepbufrDataLevelCategory')
+    lat = r.get('latitude')
+    lon = r.get('longitude')
     lon[lon>180] -= 360  # Convert Longitude from [0,360] to [-180,180]
-    sid = r.get('stationIdentification', 'prepbufrDataLevelCategory')
-    elv = r.get('stationElevation', 'prepbufrDataLevelCategory')
-    tpc = r.get('temperatureEventCode', 'prepbufrDataLevelCategory')
-    pob = r.get('pressure', 'prepbufrDataLevelCategory')
+    sid = r.get('stationIdentification')
+    sid = np.tile(sid, (lon.shape[1],1))
+    elv = r.get('stationElevation')
+    elv = np.repeat(elv, lon.shape[1])
+    elv = elv.reshape(lon.shape)
+    tpc = r.get('temperatureEventCode')
+    pob = r.get('pressure')
     pob *= 100
 
     # Time variable
-    hrdr = r.get('timeOffset', 'prepbufrDataLevelCategory')
+    hrdr = r.get('timeOffset')
     print("cycleTimeSinceEpoch")
     cycleTimeSinceEpoch = np.int64(calendar.timegm(time.strptime(date, '%Y%m%d%H%M')))
     hrdr = np.int64(hrdr*3600)
     hrdr += cycleTimeSinceEpoch
 
+    ulan = np.repeat(hrdr[:,0], hrdr.shape[1])
```

Comment:
@emilyhcliu here is the definition of the variable that is used to define the release time which is the first entry of the times array